### ISSUE TITLE
Display issues with `hector` drone in pin3

### ISSUE THREAD

**Issue by fabinsch** [stack-of-tasks/pinocchio#2291]
Since I use pinocchio3 from conda, the `hector` drone from example-robot-data is not displayed properly in meshcat. This is not blocking me, I just thought I raise the issue as maybe others will experience display issues as well. I displayed a couple other quadrupeds and manipulator models from erd and they all look fine.

Steps to reproduce:
```
conda create --name erd_debug python=3.11 pinocchio example-robot-data meshcat-python ipython
```
then in ipython
```
import example_robot_data as erd
from pinocchio.visualize import MeshcatVisualizer
robot = erd.load("hector")
vizer =  MeshcatVisualizer(
    robot.model, robot.collision_model, robot.visual_model
)
vizer.initViewer(loadModel=True, open=True)
vizer.display(robot.q0)
```

It works as expected with 
```
conda create --name erd_debug python=3.11 pinocchio==2.7.1 example-robot-data meshcat-python ipython
```
![Screenshot from 2024-06-20 09-38-59](https://github.com/stack-of-tasks/pinocchio/assets/55981657/09ba6bec-b775-4715-b585-834727bdc757)
![Screenshot from 2024-06-20 09-38-16](https://github.com/stack-of-tasks/pinocchio/assets/55981657/35f9e36c-7ede-41bb-af68-2b17bf292787)

**Comment by jorisv** [comment]
I think @ManifoldFR have found a similar issue with the UR5 end effector.

**Comment by ManifoldFR** [comment]
What is the file extension for the meshes for this robot ?

If it's STL or DAE I have a fix over on our own fork of meshcat-python on the Simple-Robotics org

**Comment by fabinsch** [comment]
You are right, the UR5 end-effector has also display issues. However, they look different than what we see on your PR @ManifoldFR .

The drone has [both](https://github.com/Gepetto/example-robot-data/blob/master/robots/hector_description/robots/quadrotor_base.urdf) DAE for visual and STL for collisions. I installed the latest master from [our fork](https://github.com/Simple-Robotics/meshcat-python) of meshcat-python but this does not solve the problem.

![image](https://github.com/stack-of-tasks/pinocchio/assets/55981657/bc137925-0cf7-407c-ae8f-e067fe9c5392)

**Comment by ManifoldFR** [comment]
Oh, the branch to install is not master, but topic/update-meshcatjs I think.

The current master on the fork is just the master from upstream if I recall correctly.

**Comment by ManifoldFR** [comment]
That one should point to the right branch on our fork of meshcat (JS) which has all of the patches I wanted to upstream. Unfortunately nothing has advanced on that front.

**Comment by fabinsch** [comment]
mmh, I tried also the branch `topic/update-meshcatjs` with commit `819526d46f9c63efad7a0f3df9a15db0cc2ce0c9` but it looks still the same.

**Comment by fabinsch** [comment]
The issue is solved for me by #2294.
